frontend/src/components/layout/Header/Header.tsx

```diff
@@ -27,6 +27,12 @@ const Header = () => {
   const network = useNetworkStore();
   const { requests } = useAppSelector(store => store.network);
 
+  // Change styling based on environment
+  let environment = 'production';
+  if (window.location.href.includes('localhost')) environment = 'local';
+  else if (window.location.href.includes('dev')) environment = 'development';
+  else if (window.location.href.includes('test')) environment = 'testing';
```

Comment:
Can we change this logic to use the environment variable NODE_ENV?

Reply:
In which case we could omit the "environment" variable completely and use process.env.NODE_ENV

Reply:
I believe you have to change the build/deploy in Openshift to do this.

Reply:
Ya the pipeline isn't really suited for that change at the moment. We would have to change any case in the code that already checks NODE_ENV as its expecting either 'production' or 'development' and wouldn't expect 'local' or 'testing'. Also if we were to change these environment variables within BuildConfigs we'd have to modify the deploy pipeline as it doesnt create a new image, it just deploys the image from the test environment.

Reply:
I'll put a ticket in the backlog to look into this. Each environment should explicitly know what environment it is